modules/persistent-storage-local-create-cr.adoc

```diff
@@ -56,7 +56,12 @@ spec:
 <4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
 <5> The file system that is created when the local volume is mounted for the first time.
 <6> The path containing a list of local storage devices to choose from.
-<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/disk/by-id/wwn`. PVs are created for these local disks when the provisioner is deployed successfully.
+<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource id, such as `/dev/disk/by-id/wwn`. PVs are created for these local disks when the provisioner is deployed successfully.
++
+[NOTE]
+====
+Local Storage Operator does not  support `LocalVolume` resource by path or partuuid, such as `/dev/disk/by-path/wwn` or `/dev/disk/by-partuuid/wwn`.
```

Comment:
```suggestion
Local Storage Operator does not support <could there be a gerund here?> the `LocalVolume` resource by path or PARTUUID, such as `/dev/disk/by-path/wwn` or `/dev/disk/by-partuuid/wwn`.
```

Also:
* "Local Storage Operator" or "The Local Storage Operator"?
* If you're using "path" or "PARTUUID" as system terms rather than generic ones, put them in backticks. 

Same comments for other note.

Reply:
Checked from aws workers, `/dev/disk/by-partuuid/wwn` should update to `/dev/disk/by-partuuid`
```
sh-4.4# cd by-partuuid/
sh-4.4# ls -lrt
total 0
lrwxrwxrwx. 1 root root 15 Mar  3 03:26 5825db80-037c-4101-998a-9e6aa95a48b5 -> ../../nvme0n1p1
lrwxrwxrwx. 1 root root 15 Mar  3 03:26 68f03c85-ae9b-4af7-af4c-aa4fee9f7089 -> ../../nvme0n1p4
lrwxrwxrwx. 1 root root 15 Mar  3 03:26 a8f11b9b-6166-4fe6-a4ef-7779959884a2 -> ../../nvme0n1p2
lrwxrwxrwx. 1 root root 15 Mar  3 03:26 8050fc56-c86b-4574-aeb8-c83a1761151d -> ../../nvme0n1p3
```

Reply:
Fixed.